### ISSUE TITLE
N1 is always fullscreen on Debian Xfce

### ISSUE THREAD

**Issue by sgosnell** [nylas/N1#1442]
N1 cannot be changed from fullscreen.  The fullscreen/restore icon is present, and changes when toggled, but the window never changes size, and cannot be resized.

Operating system: Debian Sid, with Xfce

It's always present.  Some setup screens are smaller, but the main screen is always fullscreen, not resizable.

Does this feature exist in another mail client or tool you use: All I've ever used allow switching between fullscreen and a smaller resizable window.  N1 does not, as far as I've been able to find.

**Comment by bengotow** [comment]
Hey @sgosnell — thanks for filing this. Do you know what your screen resolution is? We should be able to run N1 in a Debian VM with the same screen size to try to reproduce. It might be that your screen /is/ the minimum window size, but that seems unlikely?

**Comment by sgosnell** [comment]
1360x768 on a 26" screen.  Other screens certainly resize to smaller.  
  

Regards,

  

Stan

> On Feb 26 2016, at 12:40 pm, Ben Gotow &lt;notifications@github.com&gt;
wrote:  

>

> Hey [@sgosnell](https://github.com/sgosnell) — thanks for filing this. Do
you know what your screen resolution is? We should be able to run N1 in a
Debian VM with the same screen size to try to reproduce. It might be that your
screen /is/ the minimum window size, but that seems unlikely?

>

> —  
Reply to this email directly or [view it on GitHub](https://github.com/nylas/N
1/issues/1442#issuecomment-189418355).![](https://github.com/notifications/bea
con/AKfzNwg2UNSXJHm--8fpyg40sSehJ3Czks5poJOWgaJpZM4Hj9kg.gif)

**Comment by mbilker** [comment]
You can Toggle Full Screen from the view menu. Under `i3`, I can force an application out of full screen or into full screen mode.

**Comment by sgosnell** [comment]
I just found a partial fix for my issue.  The window was apparently too large
for the display by default.  I couldn't use the mouse to drag the screen
smaller using the right or bottom edges.  I just tried the left edge, and was
able to drag that to the right, then move the window to the left so the right
edge became visible, which I can now move.  The bottom is still unavailable,
though.  But at least I know where the problem is.   Nylas N1 seems to need
some work with the window display.  I still can't resize the window
vertically, the bottom is below my display.  I'm on a 26" monitor with
1360x768 dpi, which is the best I can get.  Every time N1 is restarted, it
comes up too large, and doesn't remember the last size.  
  

Regards,

  

Stan

> On Feb 26 2016, at 11:13 pm, Matt Bilker &lt;notifications@github.com&gt;
wrote:  

>

> You can Toggle Full Screen from the view menu. Under i3, I can force an
application out of full screen or into full screen mode.

>

> —  
Reply to this email directly or [view it on GitHub](https://github.com/nylas/N
1/issues/1442#issuecomment-189578283).![](https://github.com/notifications/bea
con/AKfzN6HuSqetsWCtOKIQYCaGuuVqVAJiks5poSfogaJpZM4Hj9kg.gif)

**Comment by sgosnell** [comment]
It seems to be working now.  I went into settings and changed my display
resolution to a larger setting which isn't usable because it's unstable
because of the limited refresh rate, and was able to resize the screen
successfully.  I think the reason it didn't remember the size was because N1
hung, and wouldn't close.   I had to use the task manager to kill it, and when
it was restarted it didn't have the proper settings.  Closing and restarting a
couple of times seems to have solved things for now.  No idea why it
originally came up so large when no other application has, but this can be
closed, I guess.  
  

Regards,

  

Stan

> On Feb 27 2016, at 9:25 am, Stan Gosnell &lt;sgosnell@riseup.net&gt; wrote:  
I just found a partial fix for my issue.  The window was apparently too large
for the display by default.  I couldn't use the mouse to drag the screen
smaller using the right or bottom edges.  I just tried the left edge, and was
able to drag that to the right, then move the window to the left so the right
edge became visible, which I can now move.  The bottom is still unavailable,
though.  But at least I know where the problem is.   Nylas N1 seems to need
some work with the window display.  I still can't resize the window
vertically, the bottom is below my display.  I'm on a 26" monitor with
1360x768 dpi, which is the best I can get.  Every time N1 is restarted, it
comes up too large, and doesn't remember the last size.  
  

>

> Regards,

>

>  

>

> Stan

>

>> On Feb 26 2016, at 11:13 pm, Matt Bilker &lt;notifications@github.com&gt;
wrote:  

>>

>> You can Toggle Full Screen from the view menu. Under i3, I can force an
application out of full screen or into full screen mode.

>>

>> —  
Reply to this email directly or [view it on GitHub](https://github.com/nylas/N
1/issues/1442#issuecomment-189578283).![](https://github.com/notifications/bea
con/AKfzN6HuSqetsWCtOKIQYCaGuuVqVAJiks5poSfogaJpZM4Hj9kg.gif)
